content/en/blog/2022-12-01-november-update/index.md

```diff
@@ -0,0 +1,379 @@
+
+- [Automate Your SLO Validation with Prometheus & Flagger - Sanskar
+   Jaiswal & Kingdon Patrick Barrett](https://youtu.be/Wgp04xTNqq4)
+
+#### GitOpsCon North America 2022
+
+- [How to Achieve (Actual) GitOps with Terraform and Flux - Priyanka
+  Ravi, Weaveworks](https://youtu.be/EXp2xAbII_k)
+- [Toward Full Adoption of GitOps and Best Practices at RingCentral -
+  Ivan Anisimov & Tamao
+  Nakahara](https://youtu.be/h8G3LM9uIHk)
+- [Simplifying Edge Deployments Using EMCO and GitOps - Igor DC &
+  Adarsh Vincent Chittilappilly,
+  Intel](https://youtu.be/cYcmXCJ2tLU)
+- [Complete DR of Stateful Workloads, PVs and CSI Snapshots via Flux
+  and Vault OSS - Kingdon
+  Barrett](https://youtu.be/jRil9H1NhZI)
+- [GitOps with Flux and OCI Registries - Soulé Ba & Scott Rigby,
+  Weaveworks](https://youtu.be/Ums3Q9kMPd8)
+- [Flux + Observability: Featuring Prometheus Operator and Pixie -
+  Somtochi Onyekwere,
+  Weaveworks](https://youtu.be/G1Mt4KE4Dao)
+
+#### KubeCon North America 2022
+
+- [Flagger, Linkerd, And Gateway API: Oh My! - Jason Morgan, Buoyant
+  & Sanskar Jaiswal,
+  Weaveworks](https://youtu.be/9Ag45POgnKw)
+- [Tutorial: How To Write a Reconciler Using K8s
+  Controller-Runtime! - Scott Rigby, Somtochi Onyekwere, Niki
+  Manoledaki & Soulé Ba, Weaveworks; Amine Hilaly, Amazon Web
+  Services](https://youtu.be/Npvz84HpO3o)
+- [Flux Maturity, Feature, and Contrib Update - Kingdon Barrett &
+  Somtochi Onyekwere,
+  Weaveworks](https://youtu.be/PhV5dJtTaDw)
+
+Here is a list of additional videos and topics we really enjoyed -
+please let us know if we missed anything of interest and we will make
+sure to mention it in the next post!
+
+### Upcoming Events 📆
+
+We are happy to announce that we have a number of events coming up in
+December- tune in to learn more about Flux and GitOps best practices,
+get to know the team and join our community.
+
+- [Nov 29: WOUG: OCI - Flux ease with helm charts and Flux (Part
+2)](https://www.meetup.com/weave-user-group/events/289767478)
+- [Nov 30: HashiCorp User Group (Luxembourg): GitOps Your Terraform
+Configurations with
+Flux](https://www.meetup.com/luxembourg-hashicorp-user-group/events/288797616)
+- [Dec 13: Implementing Flux for Scale with Soft
+Multi-tenancy](https://www.meetup.com/weave-user-group/events/289768509)
+
+The Flux project continues in active development with the addition of
+OCI configuration planned in the GA roadmap.
+Another Flux advancement has been the creation of the new VSCode
+Extension which provides a convenient interface to Flux that can help
+reduce friction moving between editor and terminal, alleviating the
+headache of context switching overloading developer focus.
+
+Flux maintainer Kingdon Barrett will demonstrate the pre-release of
+Flux's new OCI features and a convenient way to access them while they
+remain in pre-release so you can provide the feedback that is needed by
+Flux maintainers to make this feature a success!
+
+### Flux Bug Scrub
+
```

Comment:
@kingdonb Anything we should change here apart from the links and times? AND of course mention the special occasion!